src/features/accordion.js

```diff
@@ -0,0 +1,15 @@
+document.addEventListener( 'DOMContentLoaded', () => {
+	const accordions = document.querySelectorAll(
+		'.single-wmf-report .wp-block-shiro-accordion'
+	);
+
+	if ( ! accordions ) {
+		return;
+	}
+
+	Array.from( accordions ).forEach( ( accordion ) => {
+		accordion
+			.querySelector( '.accordion-item' )
+			.setAttribute( 'aria-expanded', '' );
```

Comment:
Low priority: There's no way we can set this in PHP at render...? This could cause layout jump if the accordion was at the top of the page.